fix(Popover): start implementing popover api and CSS refinements (#2369)
digdir/designsystemet: .changeset/loud-tips-return.md

---
"@digdir/designsystemet-css": patch
"@digdir/designsystemet-react": patch
---

> Popover: Rename `<Popover.Root>` to `<Popover.Context>` and use Popover API, allowing `<Popover>` to be used without `Popover.Context` as well

Reviewer: ```suggestion
Popover: 
- Rename `<Popover.Root>` to `<Popover.Context>`
- use Popover API, allowing `<Popover>` to be used without `Popover.Context`
- Remove `portal` prop
```

Author: Ou, man kan gjøre sånn! nais <3 takk, er ikke så dreven på dette formatet enda merker jeg :D
